react/src/components/ImageList.tsx

```diff
@@ -252,6 +282,15 @@ const ImageList: React.FC<{ style?: React.CSSProperties }> = ({ style }) => {
             <ConstraintTags
               tag={row?.tag}
               labels={row?.labels as { key: string; value: string }[]}
+              wrapper={(constraint: unknown) =>
+                typeof constraint === 'string' ? (
+                  <TextHighlighter keyword={imageSearch}>
+                    {constraint}
+                  </TextHighlighter>
+                ) : (
+                  <></>
+                )
+              }
```

Comment:
The TextHighlighter component needs to wrap the value, but the constraint column is using a constraint tag, so I was wondering how to extend it. I solved it by adding a wrapper attribute, but I'd like to get some feedback on what you think would be a better way.